index.html

```diff
@@ -0,0 +1,223 @@
+        <h2 class="about_myself_connect">LET'S CONNECT</h2>
+        <ul class="about_myself_social">
+          <li><a href="#"><img src="./images/github.svg" alt="github" class="github"></a></li>
+            <li><a href="#"><img src="./images/linkedinicon.svg" alt="linkedin" class="linkedin" ></a></li>
+            <li><a href="#"><img src="./images/two-finger.svg" alt="angelist" class="angelist" ></a></li>
+            <li><a href="#"><img src="./images/twitter.svg" alt="twitter" class="twitter"></a></li>
+            <li><a href="#"><img src="./images/medium.svg" alt="medium" class="medium" ></a></li>
+            
+        </ul>
+       
+        <button type="submit" class="get_resume" >
+         Get my resume</button>
+
+      <div class="list-items">
+        <ul>
+          <li><a href="#" class="languages">Languages</a></li></ul>
+            <div class="languages-vector">
+              
+            </div>
+            <div class="javascript">
+              <div class="imagejs"></div>
+              <h4 class="titlejs">JavaScript</h4>
+            
+            </div>
+
+            <div class="html">
+              <div class="imagehtml"></div>
+              <h4 class="titlehtml">HTML</h4>
+            
+            </div>
+              
+            <div class="css">
+              <div class="imagecss"></div>
+              
+              <div class="titlecss">CSS</div>
+            
+            </div>
+          
+         
+          <ul><li><a href="#" class="frameworks">Frameworks</a></li></ul>
+              <div class="frameworks-vector">
+                <img src="./images/Disabled.svg" alt="disabled">
+              </div>
+           
+            <hr class="separator_top"/> 
+          <ul><li><a href="#" class="skills">Skills</a> </li></ul>
+                <div class="skills-vector">
+                  <img src="./images/Disabled.svg" alt="disabled">
+                </div>
+                <hr class="separator_bottom"/>
+              </div>
+
+
+     </section>
+  
+     <form class="contactme" action="https://formspree.io/f/mnqworab" method="POST">
+      <h2>Contact me</h2>
+  <div class="presentation_contactme">
+      <p>If you have an application you are interested in developing, 
+          a feature that you need built or a project that needs coding.
+           I’d love to help with it </p>
+  </div>
+  <ul>
+      <li>
+        <input type="text" id="name" name="user_name" maxlength = "30" placeholder="Write your name here" required>
+      </li>
+      <li>
+        <input type="email" id="mail" name="user_email" placeholder="Write your email here" required>
+      </li>
+      <li>
+        <textarea id="message" name="user_message" maxlength = "500" placeholder="Write your message here" required></textarea>
+      </li>
+      <li>
+          <button type="submit" class="button_contactme" >Get in Touch</button> 
+        </li>
+     </ul>
+     
+</form>
```

Comment:
- [ ] Kindly make sure you added all elements provided from the Figma design in your project to make it match as much as possible the template design. I noticed that you miss the **horizontal line at the bottom** and the **background icon**. 
### Figma design 
 ![bottom](https://user-images.githubusercontent.com/83042742/171521414-7bfdf39f-2a59-4ced-ad10-ea5e2fc6829b.png) 
###  Your project 
 ![sb](https://user-images.githubusercontent.com/83042742/171521541-bdc40607-a8e2-48e7-a278-493ff6cb3238.png)